core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -271,6 +277,7 @@ public function __set($strKey, $varValue)
 		switch ($strKey)
 		{
 			case 'activeRecord':
+				trigger_deprecation('contao/core-bundle', '5.0', 'The active record has been deprecated and will be removed in Contao 6. Use $dc->getCurrentRecord() instead.');
```

Comment:
You cannot `set` the current record though, so this deprecation text seems wrong?